src/models/FilterPlanter.js

```diff
@@ -33,6 +33,16 @@ export default class Filter {
       where.organizationId = this.organizationId;
     }
 
+    if (this.email) {
+      where.email = {
+        ilike: this.email,
+      };
+    }
+
+    if (this.phone) {
+      where.phone = this.phone;
```

Comment:
I think this should use `ilike` as well for consistency with the other string filters.

Reply:
Great point